integrations/velero/documentation.yaml

```diff
@@ -0,0 +1,91 @@
+exporter_type: included
+app_name_short: Velero
+app_name: {{app_name_short}}
+app_site_name: Velero
+app_site_url: https://velero.io/
+exporter_name: the elero exporter
+exporter_pkg_name: velero
+exporter_repo_url: https://velero.io/docs/
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. You can run the following checks to verify
+  that {{app_name_short}} is exposing Prometheus-format metrics.
+
+   *  To determine if {{app_name_short}} is exposing metrics, run the following command:
+
+      <pre class="devsite-click-to-copy">
+      kubectl -n velero port-forward deploy/velero 8085
+      curl localhost:8085/metrics
+      </pre>
+dashboard_available: true
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: velero
+    labels:
+      app.kubernetes.io/name: velero
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+  + - port: metrics
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+  +     deploy: velero
+additional_podmonitoring_info: |
+  Ensure that the port and matchLabels match that of the {{app_name_short}} pods you wish to monitor.
+  {{app_name_short}} installations include the `metrics` named port and label `deploy: velero` by default.
```

Comment:
This is really hard to parse. I think it means something like this:

include a \`port\' value of \`metrics'` and a \`deploy\` label with the value \`velero\`

It might be clearer to say "By default, {{app_name_short}} installations include a \`port\' value of \`metrics'` and a \`deploy`\l abel with the value \`velero\`."

Reply:
Hmmm, my only slight pushback to @mtabasko is that we typically say "X is listening on port 8085" or "X exposes the `metrics` port", never "X has a `port` value of `metrics`"

- "By default, {{app_name_short}} installations have a port named `metrics`"
- "By default, {{app_name_short}} installations expose metrics on a port named `metrics`"

sounds more natural. But I don't feel super strongly about it.

Reply:
OK I see that we're using this phrasing e.g. in the Jenkins PR too (https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/425/commits/5c3e15f8b47c4bf5cdcd1c5daa2bf6253839bb70).

In that case, I don't mind keeping this as it is, and then if we do decide to tweak the wording, it's less churn to push something to all the affected configs at once as a follow-up